src/components/UniversalNavbar/UniversalNavbar.js

```diff
@@ -99,6 +107,73 @@ class UniversalNavbar extends React.Component {
       </div>
     )
 
+    const renderSearchIcon = (link) => (
```

Comment:
Oh wow, interesting timing on this one. So, we already have a `SearchInput` primitive, and I'm actually about to merge https://github.com/getethos/ethos-design-system/pull/248 which adds `AsyncTypehead`. This all is quite similar to the async version of react-bootstrap-typeahead. Also, we now have font awesome Pro account that's available to us and so I suspect these inline SVG's should be favoring those unless there's something ultra custom by design and we should also be coordinating with Design on if we can find a similar enough icon off fontawesome's site. I think their keen to use that as well so you shouldn't get push back.

Reply:
Update: Merged EDS [PR 248](https://github.com/getethos/ethos-design-system/pull/248)

Please pull that down and have a look. I've tried to supply a lot of guidance in those docs, so please read through those first. Pretty sure we can use that in place of `renderSearchIcon` and the search field you have here.

Reply:
Re: Icons - yes I'm going to speak to design today and see if they have time to update their designs to use FontAwesome or if we can just do a straight swap with these SVGs!

Re: AsyncTypeahead - I'll have a read through. If it's not too much work I'll try to integrate now. However, if we can bear with the current style of things in UniversalNavbar for a short time, there is a complete overhaul to the Navbar that I'm starting on by next week at the latest. I could definitely work to incorporate AsyncTypeahead usage then. [Asana ticket](https://app.asana.com/0/1116481661798430/1158569544825133/f)